.github/workflows/build-test-sources.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
     strategy:
       matrix:
         plan:
-        - rust: 1.73
+        - rust: 1.82
```

Comment:
I'm wondering, is it necessary to test that we can build all this with the MSRV as well? Or is it enough that it is specified in `cargo.toml`?

Reply:
You're right. The `check-base-compiles` job is supposed to do this, so I'll set that to the MSRV.